### PR TITLE
bpo-35641: IDLE - calltip not properly formatted if no docstring

### PR DESCRIPTION
**Pull request by eamanu**
[bpo-35641](https://bugs.python.org/issue35641): IDLE: calltips not properly formatted for functions without doc-strings

https://bugs.python.org/issue35641

### CODE REVIEW

File: Lib/idlelib/calltip.py
Comment (remilapeyre) [inline Lib/idlelib/calltip.py]: You forgot a debug statement.

Comment (taleinat) [inline Lib/idlelib/calltip.py]: This needs to be removed.

**Review by taleinat — CHANGES_REQUESTED** [review]
Other than my inline comment, two more things:

1. Add a test for this bug. It should fail without the fix and pass with the fix.
2. Add a NEWS entry. The easiest way to do so is to follow the "Details" link next to the failing "bedevere/news" check near the bottom of this page, which will lead you to the "blurb-it" tool.

**Review by remilapeyre — CHANGES_REQUESTED** [review]
Thanks for your contribution!

When you write a commit message, make it more descriptive about what the commit does, it's very important when diving into git log, here are some tips about commit messages: https://devguide.python.org/pullrequest/#making-good-commits

You need to use `blurb add` or create manually an entry to `Misc/NEWS`.

Please add some test to your pull request :)

File: Lib/idlelib/calltip.py
Comment (eamanu) [reply Lib/idlelib/calltip.py]: holy .....  sorry!!!

Comment (terryjreedy) [inline Lib/idlelib/calltip.py]:
```suggestion
```
This is a bug.

**Comment by eamanu** [comment]
Thanks for your comments. I will push tests

File: Lib/idlelib/calltip.py
Comment (terryjreedy) [reply Lib/idlelib/calltip.py]: When I hit Commit suggestion, GitHub said "Not authorized to edit …"  even though I am, and did with pencil icon.  I gave them feedback.

Comment (terryjreedy) [reply Lib/idlelib/calltip.py]: I did it with pencil edit.

**Review by terryjreedy — CHANGES_REQUESTED** [review]
What tal said.  Add at least one test *and* a blurb file.  I will likely edit whatever you write before merging, so don't agonize over the wording.  But with a file, I can do so in the online pencil editor.

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
Tal, since our change requests were identical, I will merge this when CI passes.

**Comment by taleinat** [comment]
Sorry for the noise; both Terry and I added the "needs backport to 3.7", which caused the second backport attempt to fail.

**Comment by terryjreedy** [comment]
Better than neither of us ;-).  Part of the problem was that the robots + GitHub were *really* slow in posting anything, so I removed and re-added the label after the backport has been done.

**Comment by terryjreedy** [comment]
I just noticed that the blurb was put in the Library directory instead of the Idle directory.  #11479 moves it.

**Pull request merged**
